ml-fundamentals/ml-feasibility-study.md

```diff
@@ -0,0 +1,98 @@
+# ML Feasibility Studies
+
+The main goal of Machine Learning (ML) feasibility studies is to assess whether the problem in question is feasible to solve satisfactorily using ML with the available data. They work to mitigate the potential for over-investment before there exists:
```

Comment:
```suggestion
The main goal of Machine Learning (ML) feasibility studies is to assess whether it is feasible to solve the problem satisfactorily using ML with the available data. We want to avoid investing too much in the solution before we have:
```

Reply:
Very nice!